### ISSUE TITLE
Problems when passing table.appendRows around (v2)

### ISSUE THREAD

**Issue by iamEAP** [tableau/webdataconnector#40]
Finally digging into the v2 API and ran into my first issue with the SDK.

Currently at line 160, we have this method:

``` javascript
_tableau.Table = function(tableInfo, incrementValue) {
  _tableau.validateTableInfoObj(tableInfo);

  this.tableInfo = tableInfo;
  this.incrementValue = incrementValue || "";
  this.appendRows = function(data) {
    // Call back with the rows for this table
    _tableau._tableDataCallback(this.tableInfo.id, data);
  }
};
```

Unfortunately, if I pass around `table.appendRows` as a callback within my own code, I can get an error stating that I can't read property `id` of undefined as my `getData` callback is invoked.

Pretty straightforward fix:

``` javascript
_tableau.Table = function(tableInfo, incrementValue) {
  _tableau.validateTableInfoObj(tableInfo);

  this.tableInfo = tableInfo;
  this.incrementValue = incrementValue || "";
  this.appendRows = function(data) {
    // Call back with the rows for this table
    _tableau._tableDataCallback(tableInfo.id, data); // <-- Right there.
  }
};
```

**Comment by lbrendanl** [comment]
@sdesmond46, I believe this was fixed in the 2.0.8 release, can you confirm and close?

**Comment by lbrendanl** [comment]
This was fixed in 2.0.8.
